resources/views/system/importer/step1-configure.blade.php

```diff
@@ -70,14 +71,14 @@
         <td>Database Password</td>
         <td style="text-align:center;">
           <div class="form-group">
-            {{ Form::input('text', 'db_pass', null, ['class' => 'form-control']) }}
+            <input type="text" name="db_pass" class="form-control" />
           </div>
         </td>
       </tr>
 
       <tr>
         <td colspan="2" style="text-align: right;">
-          {{ Form::submit('Test Database Credentials', ['class' => 'btn btn-info', 'id' => 'dbtest_button']) }}
+          <button type="button" class="btn btn-info" id="dbtest_button">Test Database Credentials</button>
```

Comment:
Shouldn't it rather be `type="submit"` ?

Reply:
This one is used in js functions to test the database credentials, not to submit the form

Reply:
Now i saw that there is a preventdefault, fixed on the latest commit